docs/standard/base-types/best-practices.md

```diff
@@ -170,7 +170,7 @@ Often, applications pay a performance penalty for using backtracking despite the
 
 Because a word boundary is not the same as, or a subset of, a word character, there is no possibility that the regular expression engine will cross a word boundary when matching word characters. This means that for this regular expression, backtracking can never contribute to the overall success of any match -- it can only degrade performance, because the regular expression engine is forced to save its state for each successful preliminary match of a word character.
 
-If you determine that backtracking is not necessary, you can disable it by using the `(?>subexpression)` language element. The following example parses an input string by using two regular expressions. The first, `\b\p{Lu}\w*\b`, relies on backtracking. The second, `\b\p{Lu}(?>\w*)\b`, disables backtracking. As the output from the example shows, they both produce the same result.
+If you determine that backtracking is not necessary, you can disable it by using the `(?>subexpression)` language element, known as an atomic group. The following example parses an input string by using two regular expressions. The first, `\b\p{Lu}\w*\b`, relies on backtracking. The second, `\b\p{Lu}(?>\w*)\b`, disables backtracking. As the output from the example shows, they both produce the same result.
```

Comment:
This is fine and not worth changing.  But fwiw, in .NET 5 we will now automatically convert the former expression to latter.

Reply:
I agree that it is not useful to attempt to describe in the documentation when our implementation might optimize their pattern anyway.